app/src/main/java/com/capstone/event_finder/network/EventViewModel.java

```diff
@@ -23,19 +23,21 @@ public EventViewModel(@NonNull Application application) {
         eventInCache = eventInCache("");
     }
 
-    public void insert(List<Event> events) {
-        eventRepository.insert(events);
-    }
-
-    public void delete() {
+    public void clearCache() {
         eventRepository.deleteAllEvents();
     }
 
     public LiveData<List<Event>> getEvents() {
         return getEvents;
     }
 
+    public void refreshEvents() {
+        eventRepository.getEventsFromApi();
+    }
```

Comment:
💡 This is the place we'll want to coordinate between the `EventRepository` and `EventAPI` classes.  

I will leave this as an exercise for you, but consider the events you want to take place when we explicitly want to refresh events:

1) Call the API
2) Store the results in the repository
3) Update the live data list with the latest values from the repository